hera_pspec/conversions.py

```diff
@@ -286,6 +292,51 @@ def X2Y(self, z, little_h=True):
         """
         return self.dRperp_dtheta(z, little_h=little_h)**2 * self.dRpara_df(z, little_h=little_h)
 
+    def bl_to_kperp(self, z, little_h=True):
+        """
+        Conversion factor from baseline length [meters] to k_perpendicular mode [h Mpc-1] at a specified redshift.
+
+        Parameters
+        ----------
+        z : float, redshift at which to perform calculation
+
+        little_h : boolean, optional
+            Whether to have cosmological length units be h^-1 Mpc or Mpc
+            Default: h^-1 Mpc
+
+        Return
+        ------
+        bl2kpara : float, conversion factor in units [h Mpc-1 / meters]
+        """
+        # Parsons 2012, Pober 2014, Kohn 2018
+        bl2kpara = 2*np.pi / (self.dRperp_dtheta(z, little_h=little_h) * (units.c / self.z2f(z)))
```

Comment:
This says `kpara`, but aren't we doing `kperp`?

Reply:
good catch! thanks